snippets/hamburger-button.md

```diff
@@ -6,59 +6,59 @@ tags: interactivity,beginner
 This is a way to build simple hamburger button for menu bar.
 
 ```html
-<button class="hb"></button>
+<div class="hamburger-menu">
+  <div class="top"></div>
+  <div class="middle"></div>
+  <div class="bottom"></div>
```

Comment:
These are missing `bar` class based on your code

Reply:
Sorry about that. Added the missing classes